### PR TITLE
Lesson 3

### PR DESCRIPTION
**Pull request by Akamka**

### CODE REVIEW

File: homework_3/homework3.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#1
+i = " I Love Python"
+print(i*42)
+#2
+age_in_month = 12*17
+print("month =", age_in_month) 
+#3
+age_in_years = 17
```
Comment (klymenok) [inline homework_3/homework3.py:8]: This value should be calculated based on `age_in_month` value

File: homework_3/homework3.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+#4
+my_age = "Му name is Dmytro and I’m 17 years old"
```
Comment (klymenok) [inline homework_3/homework3.py:10]: Here you should use string concatenation to put `17` not as a raw value but as a value from the variable `age_in_years`. You can user simple concatenation ("some" + "thing" + some_variable), or f-strings

File: homework_3/homework3.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+v = (comparison !=17)
+
+print(z,x,l,v)
+
+#6
+a=2
+b=5
+c=6
+
+d=(2*5+6)**2
```
Comment (klymenok) [inline homework_3/homework3.py:25]: The final value should be a string. To create a string from integers you can transform integers to strings using str() function and concatenate them using `+` sign

File: lesson_3.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
+#1
```
Comment (klymenok) [inline lesson_3.py:1]: This file is duplicated. Remove it, please

**Review by klymenok — CHANGES_REQUESTED** [review]
There are a few mistakes, fit them, please

File: homework_3/homework3.py (hunk 1 above)
Comment (Akamka) [reply homework_3/homework3.py]: І redid this work. Іs everything ok now?

Comment (klymenok) [reply homework_3/homework3.py]: To calculate this do not use raw value. You can use a value stored in `age_in_month`, like this:
`age_in_years  = int(age_in_month / 12)`

**Pull request merged**